monitoring/.env.dev.example

```
@@ -0,0 +1,8 @@
+DEPLOY_ENV=dev
+HOST_LABEL=houme-dev
+GRAFANA_BIND=127.0.0.1
+PROMETHEUS_BIND=127.0.0.1
+LOKI_BIND=127.0.0.1
+PROMETHEUS_RETENTION=15d
+GRAFANA_ADMIN_USER=admin
+GRAFANA_ADMIN_PASSWORD=change-me
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_PASSWORD key should go before the GRAFANA_ADMIN_USER key

(UnorderedKey)